embassy-rp/src/pwm.rs

```diff
@@ -93,6 +93,7 @@ impl<'d, T: Channel> Pwm<'d, T> {
         inner: impl Peripheral<P = T> + 'd,
         a: Option<PeripheralRef<'d, AnyPin>>,
         b: Option<PeripheralRef<'d, AnyPin>>,
+        b_pull: Pull,
```

Comment:
could you make this a field in `Config`? this way the number of params doesn't explode and it doesn't increase verbosity for users that don't need the pull.

(yes, divmode should've been a field in config too...)

Reply:
actually no, it's better to have it separate so it's only present in the `new_` constructots that use inputs, since it doesnt' make sense for outputs. 

(same for `divmode`, that's why it was that way...)